examples/with-rollup/.gitignore

```diff
@@ -0,0 +1,32 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+node_modules
+.pnp
+.pnp.js
+
+# testing
+coverage
+
+# next.js
+.next/
+out/
+build
+
+# misc
+.DS_Store
+*.pem
```

Comment:
Is this a copy paste or are there actually any docs or workflows that generate certs somewhere?

Reply:
This is a typical copy paste in a JS/TS .gitignore. At least I'm used to seeing it anyway. 😄